internal/agent/install.go

```diff
@@ -164,6 +167,7 @@ func Install(sourceImgURL string, dir ...string) error {
 
 	if tk != "" {
 		qr.Print(tk)
+		displayInfo(agentConfig)
```

Comment:
I can't follow the code. When does this happen?

Reply:
on images with provider during boot it triggers the install and shows the QR code and waits for an event. Its weird, yeah